### DISCUSSION TITLE
Flashing the AI deck with a USB blaster

### DISCUSSION THREAD

**Discussion by peihan-li** [bitcraze/discussions#672] [Q&A]
Hi, I am trying to make the AI deck work now. The deck I got is AI deck 1.1 rev. C, which I learned that I should have a JTAG to flash the bootloader. The only JTAG enabled (at least was saying so on the product description) is the [USB Blaster 2](https://www.amazon.com/gp/product/B08CV3L6LF/ref=ewc_pr_img_2?smid=A2SA28G0M1VPHD&psc=1). Can this can do the same trick as Olimex ARM-USB-TINY-H JTAG?

**Comment by ataffanel** [comment]
It may work as long as it is supported by OpenOCD.

In our test, the Olimex and J-Link debug probes worked. We have never tested an USB Blaster so if you already have it you can test and please report if it works. Otherwise I would suggest sourcing the Olimex or a J-Link probe instead.
